app-typescript/components/DateTimePicker.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react';
+import {DatePicker} from '../components/DatePicker';
+import {Spacer} from '@superdesk/common';
+import {defaultTo, padStart} from 'lodash';
+import {TimePicker} from './TimePicker';
+import {IconButton} from './IconButton';
+
+interface IProps {
+    value: Date | null;
+    label: string;
+    dateFormat: string;
+    onChange: (value?: string) => void;
```

Comment:
value in `onChange` shouldn't be undefined. Maybe `string | null` ? Even if it'd be undefined, it'd be better to type `value: string | undefined`